coffea/processor/executor.py

```diff
@@ -595,37 +668,46 @@ def work_queue_executor(items, function, accumulator, **kwargs):
 
         infile_function = os.path.join(tmpdir, "function.p")
 
-        # Nothing to do?  Just return the accumulator.
-        if len(items) == 0:
-            return accumulator
-
         # Keep track of total tasks in each state.
         tasks_submitted = 0
         tasks_done = 0
-        tasks_total = len(items)
+        items_submitted = 0
+        items_done = 0
 
         print("Listening for work queue workers on port {}...".format(_wq_queue.port))
 
         # Create a dual progress bar to show submission and completion.
```

Comment:
Since the progress bars are now counting events rather than chunks, I expected to see changes to `submit_bar.update` and `complete_bar.update`.